docker-compose.yml

```diff
@@ -29,8 +29,7 @@ services:
     volumes:
       - ./pretix.cfg:/etc/pretix/pretix.cfg:ro
       - pretix-data:/data
-      - pretix-src:/pretix/src
-  
+
```

Comment:
sorry, i missed this question. (for some reason i am not getting notifications on replies to my pull requests)
i don't understand what you are asking though. which volume do we not need?

in any case i am surprised that this is not merged yet. i believe this is a critical fix. without it we always risk testing the wrong code instead of the actual docker container.